e2e/utils/test-utils.ts

```diff
@@ -14,11 +14,12 @@ import {WorkspaceAccessLevel} from 'app/text-labels';
 import WorkspacesPage from 'app/page/workspaces-page';
 import {makeWorkspaceName} from './str-utils';
 
+
 export async function signIn(page: Page, userId?: string, passwd?: string): Promise<void> {
   const loginPage = new GoogleLoginPage(page);
   await loginPage.login(userId, passwd);
   // this element exists in DOM after user has logged in
-  await page.waitForFunction(() => !!document.querySelector('app-signed-in'));
+  await page.waitForFunction(() => !!document.querySelector('app-signed-in'), {timeout: 60000});
```

Comment:
this still seems high for signin. I feel like if it's  going to succeed, it'll do so in 10 seconds.

Reply:
Yes for single login session. But when Puppeteer tests are running in CI, up to 12 browsers hitting server simultaneously. In reality, unfortunately, speed isn't great.

Reply:
The legitimate issue that's being mitigated here is that our servers are very slow on initial start up, and we're constantly pushing new versions to test. Without building some leniency into the test process, we're likely to see frequent failures from that. Reducing the server startup time or smoothing out the traffic shift on deployment are all reasonable mitigations to this that may be worth pursuing.